.github/workflows/enonic-docgen.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - "master"
+      - "guide-update"
```

Comment:
@reisfmb Will we need guide-update in master?

Reply:
I believe we'll not need that when this is merged :+1: